### PR TITLE
20 profile All set

### PR DESCRIPTION
**Pull request by Allisonc369**
All parts related to profile were completed.

### CODE REVIEW

File: src/main/java/entities/Profile.java [hunk 1]
```diff
@@ -13,82 +16,144 @@ public class Profile implements Serializable {
...
         }
-        return result;
+        return result.toString();
     }
 
+    /**
+     * A method that returns all information of the user in this profile.
+     * @return Return a string that contains the age, height, weight, and gender of the user in this profile.
+     */
     public String soutInfo(){
```
Comment (Hclover2003) [inline src/main/java/entities/Profile.java:152]: what does sout mean/stand for?

File: src/main/java/profile/ProfileOutputBoundary.java [hunk 2]
```diff
@@ -1,13 +1,65 @@
...
+    void setUI(ProfileBox profileBox);
+
+    /**
+     * Set up a InfoSetBox interface.
+     * @param infoSetBox A InfoSetBox interface.
+     */
+    void setUI(InfoSetBox infoSetBox);
+
+    /**
+     * Set up a InfoViewBox interface.
```
Comment (Hclover2003) [inline src/main/java/profile/ProfileOutputBoundary.java:61]: nice overloading of method here :)

**Review by Hclover2003 — DISMISSED** [review]
looks good, added lots of javadoc and tests

File: src/main/java/entities/Profile.java (hunk 1 above)
Comment (Allisonc369) [reply src/main/java/entities/Profile.java:152]: to print out all information of the user in the profile.

**Review by luciatan — APPROVED** [review]

**Review by jkahn506 — APPROVED** [review]
Looks good.

**Review by Hclover2003 — APPROVED** [review]
still looks good

**Review by Brenden-Yiping-Wang — DISMISSED** [review]
Good! Javadoc helps reader understand the code more clearly.

**Review by jkahn506 — DISMISSED** [review]
Looks good

**Review by Brenden-Yiping-Wang — APPROVED** [review]
Good

**Review by luciatan — APPROVED** [review]

**Review by Hclover2003 — APPROVED** [review]

**Pull request merged**